### ISSUE TITLE
Cannot launch VS with error

### ISSUE THREAD

**Issue by xiaofeng0123** [microsoft/PTVS#5568]
Build of Visual Studio:16.3.0 Preview 3.0 [29211.14.master]

Repro step:
1. Install VS
2. Launch VS

Actual Result:
Cannot launch VS with error
![image](https://user-images.githubusercontent.com/41426540/62847361-7d3c4280-bd08-11e9-9525-eef9c001be04.png)

Expect Result:
VS should be launched successfully

**Comment by huguesv** [comment]
Happens on 16.3.0 Preview 3.0 [29211.14.master] as well.


```
>	Microsoft.VisualStudio.Shell.Connected.dll!Microsoft.VisualStudio.Shell.Connected.UserInformation.FirstLaunchSignIn.FirstLaunchSignIn_Loaded(object sender, System.Windows.RoutedEventArgs e)	Unknown
 	PresentationCore.dll!System.Windows.RoutedEventHandlerInfo.InvokeHandler(object target, System.Windows.RoutedEventArgs routedEventArgs)	Unknown
 	PresentationCore.dll!System.Windows.EventRoute.InvokeHandlersImpl(object source, System.Windows.RoutedEventArgs args, bool reRaised)	Unknown
 	PresentationCore.dll!System.Windows.UIElement.RaiseEventImpl(System.Windows.DependencyObject sender, System.Windows.RoutedEventArgs args)	Unknown
 	PresentationCore.dll!System.Windows.UIElement.RaiseEvent(System.Windows.RoutedEventArgs e)	Unknown
 	PresentationFramework.dll!System.Windows.BroadcastEventHelper.BroadcastEvent(System.Windows.DependencyObject root, System.Windows.RoutedEvent routedEvent)	Unknown
 	PresentationFramework.dll!System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(object root)	Unknown
 	PresentationCore.dll!MS.Internal.LoadedOrUnloadedOperation.DoWork()	Unknown
 	PresentationCore.dll!System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()	Unknown
 	PresentationCore.dll!System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()	Unknown
 	PresentationCore.dll!System.Windows.Media.MediaContext.RenderMessageHandlerCore(object resizedCompositionTarget)	Unknown
 	PresentationCore.dll!System.Windows.Media.MediaContext.RenderMessageHandler(object resizedCompositionTarget)	Unknown
 	PresentationCore.dll!System.Windows.Media.MediaContext.Resize(System.Windows.Media.ICompositionTarget resizedCompositionTarget)	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndTarget.OnResize()	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndTarget.HandleMessage(MS.Internal.Interop.WindowMessage msg, System.IntPtr wparam, System.IntPtr lparam)	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndSource.HwndTargetFilterMessage(System.IntPtr hwnd, int msg, System.IntPtr wParam, System.IntPtr lParam, ref bool handled)	Unknown
 	WindowsBase.dll!MS.Win32.HwndWrapper.WndProc(System.IntPtr hwnd, int msg, System.IntPtr wParam, System.IntPtr lParam, ref bool handled)	Unknown
 	WindowsBase.dll!MS.Win32.HwndSubclass.DispatcherCallbackOperation(object o)	Unknown
 	WindowsBase.dll!System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate callback, object args, int numArgs)	Unknown
 	WindowsBase.dll!System.Windows.Threading.ExceptionWrapper.TryCatchWhen(object source, System.Delegate callback, object args, int numArgs, System.Delegate catchHandler)	Unknown
 	WindowsBase.dll!System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority priority, System.TimeSpan timeout, System.Delegate method, object args, int numArgs)	Unknown
 	WindowsBase.dll!MS.Win32.HwndSubclass.SubclassWndProc(System.IntPtr hwnd, int msg, System.IntPtr wParam, System.IntPtr lParam)	Unknown
 	[Native to Managed Transition]	
 	[Managed to Native Transition]	
 	WindowsBase.dll!MS.Win32.HwndSubclass.DefWndProcWrapper(System.IntPtr hwnd, int msg, System.IntPtr wParam, System.IntPtr lParam)	Unknown
 	[Native to Managed Transition]	
 	[Managed to Native Transition]	
 	WindowsBase.dll!MS.Win32.HwndSubclass.SubclassWndProc(System.IntPtr hwnd, int msg, System.IntPtr wParam, System.IntPtr lParam)	Unknown
 	[Native to Managed Transition]	
 	[Managed to Native Transition]	
 	PresentationCore.dll!System.Windows.Interop.HwndSource.Resize(System.Windows.Size newSize)	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndSource.OnLayoutUpdated(object obj, System.EventArgs args)	Unknown
 	PresentationCore.dll!System.Windows.ContextLayoutManager.fireLayoutUpdateEvent()	Unknown
 	PresentationCore.dll!System.Windows.ContextLayoutManager.UpdateLayout()	Unknown
 	PresentationCore.dll!System.Windows.UIElement.UpdateLayout()	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndSource.SetLayoutSize()	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndSource.RootVisualInternal.set(System.Windows.Media.Visual value)	Unknown
 	PresentationCore.dll!System.Windows.Interop.HwndSource.RootVisual.set(System.Windows.Media.Visual value)	Unknown
 	PresentationFramework.dll!System.Windows.Window.SetRootVisual()	Unknown
 	PresentationFramework.dll!System.Windows.Window.SetRootVisualAndUpdateSTC()	Unknown
 	PresentationFramework.dll!System.Windows.Window.SetupInitialState(double requestedTop, double requestedLeft, double requestedWidth, double requestedHeight)	Unknown
 	PresentationFramework.dll!System.Windows.Window.CreateSourceWindow(bool duringShow)	Unknown
 	PresentationFramework.dll!System.Windows.Window.CreateSourceWindowDuringShow()	Unknown
 	PresentationFramework.dll!System.Windows.Window.SafeCreateWindowDuringShow()	Unknown
 	PresentationFramework.dll!System.Windows.Window.ShowHelper(object booleanBox)	Unknown
 	PresentationFramework.dll!System.Windows.Window.Show()	Unknown
 	PresentationFramework.dll!System.Windows.Window.ShowDialog()	Unknown
 	Microsoft.VisualStudio.Shell.15.0.dll!Microsoft.Internal.VisualStudio.PlatformUI.WindowHelper.ShowModal(System.Windows.Window window, System.IntPtr parent)	Unknown
 	Microsoft.VisualStudio.Shell.15.0.dll!Microsoft.Internal.VisualStudio.PlatformUI.WindowHelper.ShowModalElement(Microsoft.VisualStudio.Shell.Interop.IVsUIElement element, System.IntPtr parent)	Unknown
 	Microsoft.VisualStudio.Shell.UI.Internal.dll!Microsoft.VisualStudio.Services.SignIn.SignInWizardService.ShowWizard(Microsoft.VisualStudio.Shell.Interop.IVsUIElement element)	Unknown

```

**Comment by huguesv** [comment]
Should be fixed in 29212.15
